inference-engine/tests/unit/gna/ngraph/transformations/gna_pwl.cpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "transformations/transpose_to_pwl.hpp"
+
+#include "common_test_utils/data_utils.hpp"
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include <ngraph/function.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/pass/manager.hpp>
+#include <ngraph/pass/serialize.hpp>
+#include <transformations/init_node_info.hpp>
+
+namespace pwl_test {
+std::shared_ptr<ngraph::Function> CreateSigmoid(const ngraph::Shape& input_shape) {
+    auto input_params = std::make_shared<ngraph::opset1::Parameter>(ngraph::element::f32, input_shape);
+    auto sigmoid = std::make_shared<ngraph::opset1::Sigmoid>(input_params);
+    auto result = std::make_shared<ngraph::opset1::Result>(sigmoid);
+    return std::make_shared<ngraph::Function>(ngraph::ResultVector{result}, ngraph::ParameterVector{input_params});
+}
+
+std::shared_ptr<ngraph::Function> CreateTanh(const ngraph::Shape& input_shape) {
```

Comment:
Looks like we can create the template function for all PWL functions and parameterize the tests

Reply:
done